### DISCUSSION TITLE
How to remove "underline" from TextBox

### DISCUSSION THREAD

**Discussion by BestClaws** [bdlukaa/fluent_ui#626] [Q&A] [answered]
I would like to remove the underline that the textbox has, I tried adding/removing borders, but the underline doesn't appear to be a border, as adding a border only wraps around it.

![image](https://user-images.githubusercontent.com/40067018/204407770-454dea41-285c-436b-a0ac-e3696d5ee17d.png)

**Comment by bdlukaa** [comment, marked as answer]
You can provide the `foregroundDecoration` property to make updates to the foreground of the text box. By default, it handles the foreground when focused, disabled or enabled

**Reply by thegioiso** [reply]
@bdlukaa can i change this underline color when it enabled? It only has `highlightColor`  in AutoSuggestBox widget. My bad, I dont know how to custom it in `foregroundDecoration`...

**Reply by robkri** [reply]
> You can provide the `foregroundDecoration` property to make updates to the foreground of the text box. By default, it handles the foreground when focused, disabled or enabled
https://github.com/bdlukaa/fluent_ui/discussions/626#discussioncomment-4267754

As I was curious how this works, I tried it and came up with this: 

```
TextBox(
  foregroundDecoration: BoxDecoration(
    border: Border(
      bottom: BorderSide.none,
    ),
  ),
)
```

**Reply by robkri** [reply]
> can i change this underline color when it enabled? It only has `highlightColor` in AutoSuggestBox widget. My bad, I dont know how to custom it in `foregroundDecoration`...

If you want to change the color the `AutoSuggestBox` has when selected, you have to use `highlightColor`. If you want to change the color of the underline when not selected, you're right: There seems to be no matching property. `TextBox` has `highlightColor` _as well as_ `unfocusedColor`, whereas `AutoSuggestBox` only has `highlightColor`.
